src/frontend/ui/établissement-territorial-sanitaire/InfoBulle/ContenuReconnaissancesContractuelles.tsx

```diff
@@ -0,0 +1,58 @@
+import { ReactChild } from 'react'
+
+import { useDependencies } from '../../commun/contexts/useDependencies'
+
+type ContenuReconnaissancesContractuellesProps = Readonly<{
+  dateDeMiseÀJour: string
+  source: ReactChild
+}>
+
+export const ContenuReconnaissancesContractuelles = ({ dateDeMiseÀJour, source }: ContenuReconnaissancesContractuellesProps) => {
+  const { wording } = useDependencies()
+
+  return (
+    <>
+      <p>
+        Mise à jour :
+        {' '}
+        {dateDeMiseÀJour}
+        {' '}
+        - Source :
+        {' '}
+        {source}
+      </p>
+      <section aria-label={wording.ÉLÉMENTS_DE_COMPRÉHENSION}>
+        <p>
+          Seules les autorisations non caduques sont transmises à FINESS par ARHGOS et sont donc publiées.
+          Hormis les autorisations en attente de première mise en œuvre, toutes les autorisations publiées peuvent donc être considérées comme actives.
+          Dans le domaine sanitaire les autorisations sont données à une entité juridique puis mises en oeuvre dans un établissement.
+          Pour indiquer les services offerts, FINESS identifie ceux-ci à l’aide d’un triplet composé des éléments suivants : Activité/Modalité/Forme.
+
+          Une activité soumise à reconnaissance contractuelle est une activité non soumise à autorisation du DG ARS mais soumise à
+          reconnaissance contractuelle dans le cadre et la durée du CPOM (contrat pluriannuel d’objectif et de moyens).

```

Comment:
C'est pas très français
```suggestion
          reconnaissance contractuelle dans le cadre et la durée du CPOM (contrat pluriannuel d’objectif et de moyens).

```